runtime/autoload/man.vim

```diff
@@ -1,6 +1,11 @@
-let s:man_tag_depth = 0
+" Ensure Vim is not recursively invoked (man-db does this)
+" More info here http://comments.gmane.org/gmane.editors.vim.devel/29085
+let s:man_cmd = 'unset MANPAGER; man 2>/dev/null '
```

Comment:
@justinmk instead of the previous code where `$MANPAGER` was emptied, this actually removes it from the environment.